### PR TITLE
Added crime and death event.

### PR DESCRIPTION
**Pull request by xiluisx**

### CODE REVIEW

File: src/Webhooks/Core.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ public class Core : Plugin
         public Webhook joinWebhook { get; set; }
         public Webhook startWebhook { get; set; }
         public Webhook commandWebhook { get; set; }
+        public Webhook deathWebhook { get; set; }
+        public Webhook crimeWebhook { get; set; }
```
Comment (ggggg) [inline src/Webhooks/Core.cs:31]: Tbh all of this should be upper case, not a big deal tho

File: src/Webhooks/Core.cs [hunk 2]
```diff
@@ -67,7 +69,7 @@ public void RegisterCustomCommands()
                     {
                         return;
                     }
-                    Logger.LogInfo($"cutstom event {customEvent.Event} was tiriggered");
```
Comment (ggggg) [inline src/Webhooks/Core.cs:70]: Yeah idk why I miss spelled it that bad lol, thanks for fixing it

**Comment by xiluisx** [comment]
> asdf

~ Asd Fgh

File: src/Webhooks/Events/OnCrime.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using BrokeProtocol.Utility;
+
+namespace Webhooks.Events
+{
+    public class OnCrime : IScript
+    {
+        [Target(GameSourceEvent.PlayerCrime, ExecutionMode.Event)]
+        public void OnEvent(ShPlayer player, byte crimeIndex, ShPlayer victim)
+        {
+            if (player.svPlayer.godMode || player.svPlayer.InvalidCrime(crimeIndex)) return;
```
Comment (ggggg) [inline src/Webhooks/Events/OnCrime.cs:12]: Check player.isHuman

File: src/Webhooks/Events/OnDeath.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        [Target(GameSourceEvent.PlayerDeath, ExecutionMode.Event)]
+        public void OnEvent(ShPlayer player, ShPlayer attacker)
+        {
+            if(attacker && attacker.isHuman && attacker!=player)
+            {
+                Core.Instance.deathWebhook.Send(string.Format(Core.Instance.Settings.General.DeathLogFormat, player.username, attacker.username), player.username, 
+                embeds: Core.Instance.Settings.General.DeathUseEmbed? EmbedCrafter.CreateAllEmbeds(Core.Instance.Settings.General.DeathEmbed,player) :null);
+            }
+        }
```
Comment (ggggg) [inline src/Webhooks/Events/OnDeath.cs]:
```suggestion
            if(!attacker || !attacker.isHuman || attacker==player)
            {
                return;
            }
            Core.Instance.deathWebhook.Send(string.Format(Core.Instance.Settings.General.DeathLogFormat, player.username, attacker.username), player.username, 
                embeds: Core.Instance.Settings.General.DeathUseEmbed? EmbedCrafter.CreateAllEmbeds(Core.Instance.Settings.General.DeathEmbed,player) :null);
        }
```

**Review by ggggg — COMMENTED** [review]
Overall good, just a few very, very minor suggestions you might want to consider.

**Comment by ggggg** [comment]
Also uhh, should codacy review this pr for me :(

**Review by ggggg — APPROVED** [review]
Great addition, thank you.

**Pull request merged**